BFit/Controller/ProfileViewController.swift

```diff
@@ -60,19 +60,29 @@ class ProfileViewController: UIViewController, UIImagePickerControllerDelegate,
         }
     }
     
-    
     func imagePickerController(_ picker: UIImagePickerController, didFinishPickingMediaWithInfo info: [UIImagePickerController.InfoKey : Any]) {
+        let imageURL = info[UIImagePickerController.InfoKey.imageURL] as! URL
         let pickedImage = info[UIImagePickerController.InfoKey.originalImage] as? UIImage
         profilePic.image = pickedImage
+        postImage(imgURL: imageURL as URL)
         dismiss(animated: true, completion: nil)
     }
     
+    func postImage(imgURL : URL) {
+        cloudinary.createUploader().upload(url: imgURL, uploadPreset: "rgflevhw")
```

Comment:
Oh right, this was something I found in looking up trying to configure Cloudinary. It's generated in the settings on our dashboard, and it's an unsigned preset, so I think it just allows more flexibility in uploading and required parameters. Not sure if it's still necessary or if it would work without it, but that's where it comes from.

Reply:
I figured as much but also see how it could have been ;alksjdf, thanks jamie!